src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -511,6 +511,34 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
             string centralPackageTransitivePinningEnabled = GetPropertySafe(_vsProjectAdapter.BuildProperties, ProjectBuildProperties.CentralPackageTransitivePinningEnabled);
             // Do not add new properties here. Use BuildProperties.GetPropertyValue instead, without DTE fallback.
 #pragma warning restore CS0618 // Type or member is obsolete
+            string skdAnalysisLevelString = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.SdkAnalysisLevel);
```

Comment:
We're missing tests for the legacy code.

Reply:
I think the review was rerequested before my latest review, so just adding this comment to make this apparent and make the nice little rerequest icon appear again :D

Reply:
I updated the code to load the property values using `ProjectRestoreMetadata.GetSdkAnalysisLevel` which has been unit tested

Reply:
I was referring to a LegacyPackageReference package spec creation test.

Something similar to https://github.com/NuGet/NuGet.Client/blob/4a1faa2bcc771862a0ee1d241ebbfa1c196630ab/test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceProjectTests.cs#L1587-L1622.